qutebrowser/config/configdata.yml

```diff
@@ -284,6 +290,39 @@ qt.low_end_device_mode:
     This improves the RAM usage of renderer processes, at the expense of
     performance.
 
+qt.chromium.sandboxing:
+  type:
+    name: String
+    valid_values:
+      - enable-all: Enable all available sandboxing mechanisms.
+      - disable-seccomp-bpf: Disable the Seccomp BPF filter sandbox (Linux only).
+      - disable-all: Disable all sandboxing (**not recommended!**).
+  default: enable-all
+  backend: QtWebEngine
+  restart: true
+  no_autoconfig: true  # due to it being dangerous
+  # yamllint disable rule:line-length
+  desc: >-
+    What sandboxing mechanisms in Chromium to use.
+
+    Chromium has various sandboxing layers, which should be enabled for normal
+    browser usage. Mainly for testing and development, it's possible to disable
+    individual sandboxing layers via this setting.
+
+    Open `chrome://sandbox` to see the current sandbox status.
+
+    Changing this setting is only recommended if you know what you're doing, as
+    it **disables one of Chromium's security layers**. To avoid sandboxing being
+    accidentally disabled persistently, this setting can only be set via
+    `config.py`, not via `:set`.
+
+    See the Chromium documentation for more details:
+
+    - https://chromium.googlesource.com/chromium/src/+/HEAD/docs/linux/sandboxing.md[Linux]
+    - https://chromium.googlesource.com/chromium/src/+/HEAD/docs/design/sandbox.md[Windows]
+    - https://chromium.googlesource.com/chromium/src/+/HEAD/docs/design/sandbox_faq.md[FAQ (Windows-centric)]
+  # yamllint enable rule:line-length
+
```

Comment:
@mschilli87 do your thing please :slightly_smiling_face:

Reply:
LGTM, but why mixing enable/disable options? AFAICT `qt.chromium.sandboxing.disable`  with options `none`/`seccomp-bpf`/`all` would provide the same choices in a more consistent way. If you decide to go for it, I'd rephrase the description of `none` to `Keep all available sandboxing mechanisms enabled`.

Reply:
I like using positive names to avoid the mental gymnastics of having a double negative. I don't remember where I picked it up, but the *Use positive, “happy” names* part of [Common API mistakes and how to avoid them - LogRocket Blog](https://blog.logrocket.com/common-api-mistakes-and-how-to-avoid-them-804fbcb9cc4b/) seems to summarize it well.

Even if a fully enabled sandbox is the obvious default, I think it's (slightly) easier to parse `sandboxing = "enable-all"` or `sandboxing = disable-all"` than it is to parse `"sandboxing.disable = "none"` or `sandboxing.disable = "all"`

Reply:
For me the idea was to 'warn' the user that all (non-default) choices for this option *decrease* security. If I were a new user and I'd see an option 'disable safety-features', I'd avoid it. If it was just named 'safety features' and contained both 'enable ...' and 'disable ...' choices, I'd be intrigued. But I understood this is a question of opinion and I can follow your argument as well. That's why I just asked instead of suggesting a change in my review in the first place. So no need to bikeshed and thanks for sharing your reasoning.